streampipes-data-explorer-commons/src/main/java/org/apache/streampipes/dataexplorer/commons/influx/InfluxStore.java

```diff
@@ -218,6 +226,21 @@ private void handleMeasurementProperty(Point.Builder p,
     }
   }
 
+  private void handleNonPrimitiveMeasurementProperty(Point.Builder p, Event event, String preparedRuntimeName) {
+    var field = event.getRaw().get(preparedRuntimeName);
+    ObjectMapper mapper = new ObjectMapper();
+    try {
+      String json = mapper.writeValueAsString(field);
+      p.addField(preparedRuntimeName, json);
+      // Deserialize in this way, tested the primitive types can be correctly deducted from Object
+//      TypeFactory typeFactory = mapper.getTypeFactory();
```

Comment:
If the code is working, I would remove this comment before we merge it. Is there a way we can unit test it? I think in the current version of this class this will be hard. But maybe we can open another issue to implement unit tests for this class and then refactor it. What do you think?

Reply:
I agree, probably we should refactor the serialization/deserialization logic into a class called `FieldSerializer`, in which the `Serialize` method takes an `AbstractField` and returns a `String` and the `Deserialize` method takes a `String` and return a `Map<string, deserialized type (as opposed to Object in my comments)>`. For deserialization, I plan to write a recursive function in which the base case is when the value is primitive or List. Do you want me to write all that logic in this PR?